src/org/labkey/test/tests/SampleTypeTest.java

```diff
@@ -241,7 +242,43 @@ public void testCreateSampleTypeNoExpression()
         sampleTypeHelper.verifyDataValues(data);
     }
 
-    // Issue 47280: LKSM: Trailing/Leading whitespace in Source name won't resolve when deriving samples
+    // Issue 53313: LKS doesn't show Sample Type fields with special characters in Custom Properties
+    @Test
+    public void testCustomProperties()
+    {
+        final String sampleTypeName = "SampleTypeCustomProps" + DOMAIN_TRICKY_CHARACTERS;
+        FieldInfo stringCol1 = new FieldInfo(TestDataGenerator.randomFieldName("StringColPlain", "\""), ColumnType.String);
+        FieldInfo stringCol2 = new FieldInfo(TestDataGenerator.randomFieldName("StringCol%"), ColumnType.String);
+        FieldInfo calcCol = new FieldInfo(TestDataGenerator.randomFieldName("CalcCol"), ColumnType.Calculation);
+        final List<FieldDefinition> fields = List.of(
+                stringCol1.getFieldDefinition(),
+                stringCol2.getFieldDefinition(),
+                calcCol.getFieldDefinition().setValueExpression("\"" + stringCol1.getName() + "\" || 'Concat'")
```

Comment:
Won't this fail if the field name includes `"`? I'll try to find a test helper to properly escape it.

Reply:
I excluded the double quote char from the randomFieldName for StringColPlain for that reason. If we want to include that char, then we just need to escape the double quote in the expression. Something like this (untested)...maybe we have a test helper for that:
```suggestion
                calcCol.getFieldDefinition().setValueExpression("\"" + stringCol1.getName().replaceAll("\"", "\"\"") + "\" || 'Concat'")
```

Reply:
Oh, I missed the exclusion. Thanks for pointing it out.

I found `EscapeUtil.getSqlQuotedValue()`. I'll switch to that and remove the exclusion